node/subsystem-util/src/runtime/mod.rs

```diff
@@ -358,13 +361,21 @@ pub enum ProspectiveParachainsMode {
 	/// v2 runtime API: no prospective parachains.
 	Disabled,
 	/// vstaging runtime API: prospective parachains.
-	Enabled,
+	Enabled {
```

Comment:
Good to see there is a distinction between allowed ancestry and depth - that'll give us some freedom in multi-core approaches to blockspace later on.

Reply:
Perhaps this should use the `AsyncBackingParameters` defined in primitives?

Reply:
1. Most of the places that use params expect it to be `usize`, and we would have to convert it from `u32` every time the value is accessed.
2. Simply looks a little less clumsy:
```rust
ProspectiveParachains::Enabled(AsyncBackingParameters { max_candidate_depth, allowed_ancestry_len })
// vs
ProspectiveParachains::Enabled { max_candidate_depth, allowed_ancestry_len }
```